packages/xstate-react/src/useSpawn.ts

```diff
@@ -10,10 +15,11 @@ import useConstant from './useConstant';
  * @returns An ActorRef with the specified `behavior`
  */
 export function useSpawn<TState, TEvent extends EventObject>(
-  behavior: Behavior<TEvent, TState>
+  behavior: Behavior<TEvent, TState>,
+  options?: SpawnBehaviorOptions
```

Comment:
Currently, this allows `parent` to be passed as part of the `options` but I think that this shouldn't be allowed. Thoughts?

Reply:
Got it.
I disagree. `Spawn` has it own interface, but disabled by a simple hook.
Now I must make another custom hook.

Reply:
I'm hesitant about allowing `parent` here because within the current system all children should be linked to their parents. It's easy to achieve that when only the parent itself can spawn an "attached" child. Allowing `parent` here would allow bypassing that and potentially leading to bizarre/corrupted states.

I have no problem with allowing the `id` option here. I would just treat `useSpawn` as always spawning a "root" actor.